packages/web/src/components/TextField/_TextField.scss

```diff
@@ -0,0 +1,100 @@
+@use 'theme';
+
+.TextField {
+    width: 100%;
+}
+
+.TextField__label {
+    display: block;
+    margin-bottom: theme.$spacing;
+    font-size: theme.$label-font-size;
+    font-weight: theme.$label-font-weight;
+    line-height: theme.$label-line-height;
+    color: theme.$label-color-default;
+}
+
+.TextField__label--hidden {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+}
+
+.TextField__label--required::after {
+    content: '*';
+    color: theme.$label-required-color;
+}
+
+.TextField__input {
+    display: block;
+    width: 100%;
```

Comment:
- Are we sure? 🤔 Is it the expected behavior? Wouldn't it be better to have a default width…?
- I think this needs to be turned off (or changed to `max-width` to be precise) for fields with `size` attribute. (Anyway, I think we will need `max-width: 100%` here so the field works inside flex and grid layouts we have yet to create.)